lisa/operating_system.py

```diff
@@ -971,10 +985,11 @@ def _update_packages(self, packages: Optional[List[str]] = None) -> None:
         self._node.execute(command, sudo=True, timeout=3600)
 
     def __verify_package_result(self, result: ExecutableResult, packages: Any) -> None:
-        # yum returns exit_code=1 if package is already installed.
-        # We do not want to fail if exit_code=1.
+        # yum returns exit_code=1 if DNF handled an error with installation.
+        # We do not want to fail if exit_code=1, but warn since something may
+        # potentially have gone wrong.
         if result.exit_code == 1:
-            self._log.debug(f"{packages} is/are already installed.")
+            self._log.warn(f"DNF handled error with installation of {packages}")
```

Comment:
What's the reason to change it to warn level? The warn level is actually not useful. Nobody check the warn even error level logs. We care about pass/faill test results, and fatal errors on test run.

Reply:
I wanted to make it easy to search for a potentially suspicious but not neccesarily fatal condition (an error code that returned reporting 'handled by backend'). In testing I learned that this can actually bite you, like with a missing package it results in none of the packages actually being installed.

In this PR I added a condition to hard fail on missing packages since it means you need to rework your test to account for the missing package, but since there are an unknown amount of other conditions people might see it seemed helpful to highlight this with a WARN label rather than just another debug level log. Does that make sense? Or you want me to switch it?

Reply:
eh nevermind I'll just move it to debug